### PR TITLE
Create Dockerfile for Kronos

### PR DESCRIPTION
**Pull request by lydiagu**
Docker has 3 kronos services, one for all, one for readonly, one for collector. When docker is run, all three are started. Three ports are mapped. The log directory is mapped to `/var/log/kronos`. Settings file at `/etc/kronos/settings.py` is mapped into the docker.

### CODE REVIEW

**Comment by lydiagu** [comment]
Made all fixes

File: kronos/scripts/install_kronosd.py [hunk 1]
```diff
@@ -56,17 +55,8 @@ def make_dirs():
...
-              '/etc/init.d/kronos')
-  with open(os.path.join(BASE_DIR, 'settings.py.template')) as f:
-    settings = f.read()
-  if not LOG_DIR_RE.search(settings):
-    raise Exception('Failed to find log directory in settings.py.template.')
-  settings = re.sub(LOG_DIR_RE, "'log_directory': '%s'" % LOG_DIR, settings)
-  with open('/etc/kronos/settings.py', 'w') as f:
-    f.write(settings)
+  kronosd_file_path = os.path.join(BASE_DIR, 'scripts/kronosd.init.d')
+  shutil.copy(kronosd_file_path, '/etc/init.d/kronosd')
```
Comment (lydiagu) [inline kronos/scripts/install_kronosd.py]: done

File: kronos/docker/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#Deploying with Docker
+
+## Quick usage HOWTO
+
+We've already pushed a working [Dockerfile to Docker Hub](https://registry.hub.docker.com/u/chronology/kronos/), so you can do something like the following:
+
+  * Put a `settings.py` in `/etc/kronos/settings.py`
+  * Copy `kronos/scripts/uwsgi.ini` to `/etc/kronos/`
```
Comment (marcua) [inline kronos/docker/README.md]: remove?

File: kronos/scripts/kronosd.init.d [hunk 3]
```diff
@@ -28,7 +28,7 @@ case "$1" in
   fi
 
 	echo -n "Starting kronosd: "
-	if (cd $LIBDIR/uwsgi && ./uwsgi --ini /etc/kronos/uwsgi.ini --pidfile $PIDFILE --daemonize $LOGFILE)
+	if (cd $LIBDIR/uwsgi && ./uwsgi --pidfile $PIDFILE --daemonize $LOGFILE)
```
Comment (marcua) [inline kronos/scripts/kronosd.init.d]: why don't you need --ini?

**Comment by lydiagu** [comment]
Also tested that uwsgi works - it does!

**Comment by marcua** [comment]
lgtm!  merge using `push_branch.sh`

**Pull request merged**